bindings/python/pydeck/docs/README.md

```diff
@@ -1,4 +1,19 @@
 pydeck API Documentation
 =========
 
-TODO This will be a table of contents for the pydeck documentation
+## Table of contents
+#### [Deck](./bindings/python/pydeck/docs/deck.md)
```

Comment:
No dot please. You are referencing an absolute path. You can click on the link if you preview this markdown.